### ISSUE TITLE
Future of this project

### ISSUE THREAD

**Issue by repat** [willdurand/nmap#34]
@willdurand:

I just installed `dev-master` and it works. However I get these 2 warnings below. 

> PHP Deprecated:  Passing a command as string when creating a "Symfony/Component/Process/Process" instance is deprecated since Symfony 4.2, pass it as an array of its arguments instead, or use the "Process::fromShellCommandline()" constructor if you need features provided by the shell. in ...vendor/symfony/process/Process.php on line 147
> PHP Deprecated:  Passing a command as string when creating a "Symfony/Component/Process/Process" instance is deprecated since Symfony 4.2, pass it as an array of its arguments instead, or use the "Process::fromShellCommandline()" constructor if you need features provided by the shell. in ...vendor/symfony/process/Process.php on line 147

I'm not sure if you want to drop support for older symfony/process or you want to somehow figure out the symfony/process version? I wasn't aware that I have write access to this repo, I made my own little fork over at github.com/repat/nmap but would be willing to maintain this instead if that's ok? (Also accidentally clicked a merge button the wrong way, but it's cleaned up ;-) - that's how I found out)

So for symfony/process ^4.0 to work without warnings, we have to pass the arguments as an array as you can see here: 

https://github.com/repat/nmap/blob/master/src/Nmap/Nmap.php#L104

I would now:
1. Create version 0.6.0 with the warnings
2. Drop support for symfony/process 2.X, only support ^3.4*
3. Pass arguments as array because that's how it's done in ^3.4 and ^4.0
4. Create version 0.7.0

Let me know what you think :-) 

*
![image](https://user-images.githubusercontent.com/516807/64508153-81b93400-d291-11e9-83df-34029be31569.png)

**Comment by willdurand** [comment]
Yeah, you can drop support for very old symfony/process versions.

**Comment by wvdongen** [comment]
@repat This fork is maintained https://github.com/DavidGoodwin/nmap/
